lib/pacemaker/pcmk_sched_colocation.c

```diff
@@ -1235,6 +1244,12 @@ pcmk__apply_coloc_to_priority(pe_resource_t *dependent,
 
     dependent->priority = pcmk__add_scores(score_multiplier * colocation->score,
                                            dependent->priority);
+    pe_rsc_trace(dependent,
+                 "Applied %s to %s promotion priority (now %s after %s %s",
```

Comment:
Missing closing parenthesis, and optionally a reason (doesn't look like there is one worth stating).

Is `dependent->priority` specifically for promotion? It's not the promotion score (see another use of the term in `pcmk__add_promotion_scores()`), and I thought priority was used for other things like assignment order. Maybe promotion is the only use from this point forward and the promotion score is equal to the priority at this point, not sure off hand.

Reply:
Yeah, priority gets repurposed for promotion purposes by pcmk__set_instance_roles() then reverted to the original priority in pcmk__create_promotable_actions()